parser/parser_system1.c

```diff
@@ -777,6 +865,27 @@ struct pv_state* system1_parse(struct pv_state *this, const char *buf)
 	// Parse full state json
 	ret = jsmnutil_parse_json(buf, &tokv, &tokc);
 
+	count = pv_json_get_key_count(buf, "disks.json", tokv, tokc);
+	if (count == 1) {
```

Comment:
maybe we could return NULL and goto out if count is different than 1?

Reply:
Idea is to keep it functional even when disks.json isn't added to allow the existing setup.